### PR TITLE
Fixed Hebrew Language

### PR DESCRIPTION
**Pull request by Flaysh**
Hey there! I use DarkReader for more than 3 years and pay for the iOS app.
I always wanted to contribute, guess it's time.

- I fixed/improved the Hebrew language.
- Added 2 websites to the list.

### CODE REVIEW

**Comment by Gusted** [comment]
Hi @Flaysh!

Thank you for your contribution, could please split this PR? So it will show up nicely in the commit graph on the main branch once merged. Please make a separate PR for the added websites and remove them from this PR.

**Comment by Flaysh** [comment]
Hey, Just did.
New PR on #9819 

Thank u!

**Review by Gusted — APPROVED** [review]
Many thanks!

**Review by Gusted — CHANGES_REQUESTED** [review]
Please change your PR to match the same amount of lines for each translation key.

File: src/_locales/he.config [hunk 1]
```diff
@@ -28,7 +28,7 @@
 הגדר מקש קיצור
 
 @automation
-הפעלה אוטומטית:
+הפעלה אוטומטית
```
Comment (Gusted) [inline src/_locales/he.config:31]: Trailing new line here.

File: src/_locales/he.config [hunk 2]
```diff
@@ -170,16 +170,16 @@
 @open_dev_tools
 Dev tools
 
-
 #==== Store listing ====
 
+
```
Comment (Gusted) [inline src/_locales/he.config:175]:
```suggestion
```

File: src/_locales/he.config [hunk 3]
```diff
@@ -170,16 +170,16 @@
 @open_dev_tools
 Dev tools
```
Comment (Gusted) [inline src/_locales/he.config:172]:
```suggestion


```

**Review by Gusted — APPROVED** [review]

**Pull request merged**